spec/index.bs

```diff
@@ -1883,6 +1883,7 @@ enum GPUFeatureName {
     "texture-compression-astc",
     "timestamp-query",
     "indirect-first-instance",
+    "fp16-in-shader-and-buffer",
```

Comment:
We should bikeshed the feature name - I don't know what it should be, maybe just `shader-f16`? since fp16-in-buffer is only important in shaders. To be precise it's more like "f16 (variables and buffers) in shaders".

Also it matches nicely with `enable f16;`.

Reply:
Done.